src/nodejs/unit-http/http_server.js

```diff
@@ -413,7 +414,14 @@ ServerRequest.prototype._read = function _read(n) {
 };
 
 
-function Server(requestListener) {
+function Server(options, requestListener) {
+    if (typeof options === 'function') {
+        requestListener = options;
+        options = {};
+    } else {
+        stderr.write('http.Server constructor was called with unsupported options, using default settings');
```

Comment:
New line needed at the end of the error message string. Now it looks like following:

```
2024/01/30 17:14:07.682 [debug] 34852#18000878 malloc(87): 137F05980
http.Server constructor was called with unsupported options, using default settings2024/01/30 17:14:07.796 [debug] 0#18000881 [unit] Unit::Unit()
2024/01/30 17:14:07.797 [debug] 34852#18000881 [unit]
```

Reply:
Added a newline